### ISSUE TITLE
CVE-2023-38408

### ISSUE THREAD

**Issue by exploitact** [rancher/rancher#42519]
**Is your feature request related to a problem? Please describe.**
There is one CVE identified by Wiz.io to be a vulnerability.

**Describe the solution you'd like**
Updating the packages openssh-clients, openssh-fips, and openssh-common to 8.4p1-150300.3.22.1 or higher. 

**Describe alternatives you've considered**
Adding the following command to the Dockerfile: RUN zypper update openssh-clients to https://github.com/rancher/rancher/blob/release/v2.8/package/Dockerfile

**Additional context**

Currently we are using the stable image release, 
https://hub.docker.com/layers/rancher/rancher/stable/images/sha256-bf7a69196cd0af5d7a54782c29f9efd1e635ea8cc926eb6d888232b9dd5d0896?context=explore

We are currently self-hosting Rancher using AKS with deployment via Helm Charts. Our secuirty tool, Wiz.io, picked this up as a High Vulnerability.

---

Wiz.io findings

The package openssh-clients version 8.4p1-150300.3.18.2 was detected in ZYPPER package manager on a container image running Suse 15.4 is vulnerable to CVE-2023-38408, which exists in versions < 8.4p1-150300.3.22.1.

The vulnerability was found in the Official Suse Security Advisories with vendor severity: High (NVD severity: Critical).

This vulnerability has a known exploit available. Source: Packetstorm.

The vulnerability can be remediated by updating the package to version 8.4p1-150300.3.22.1 or higher, by adding the following command to the Dockerfile: RUN zypper update openssh-clients.

**Comment by pjbgf** [comment]
From what I can see, this CVE is already patched at `v2.7` head branch and it will be fixed on the `stable` tag once the next release is out. Here's what Trivy is saying for CVEs at the `rancher/rancher:v2.7-head` base image:

```
rancher/rancher:v2.7-head (suse linux enterprise server 15.5)
=============================================================
Total: 0 (UNKNOWN: 0, LOW: 0, MEDIUM: 0, HIGH: 0, CRITICAL: 0)
```

**Comment by exploitact** [comment]
Hi @pjbgf thank you so much for your answer. Do you know when the next release is going to happen?
